Data/Boundaries/Smoothing/Omaio_Bay1_v22p5

```diff
@@ -0,0 +1,187 @@
+177.61612328 -37.82127620

```

Comment:
This is the old smoothing line file. Is it still meant to be here or should it be removed? I'm not sure if it's interfering.